website/docs/r/kusto_cluster.html.markdown

```diff
@@ -92,6 +94,18 @@ An `identity` block supports the following:
 
 ~> **NOTE:** When `type` is set to `SystemAssigned`, the Principal ID can be retrieved after the cluster has been created. More details are available below. See [documentation](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview) for additional information.
 
+---
+
+A `optimized_auto_scale` block supports the following:
+
+* `version` - (Computed) The version of the template defined, for instance 1.
+
+* `enabled` - (Computed) A boolean value that indicate if the optimized autoscale feature is enabled or not.
```

Comment:
these are computed - so if they're not being user-configurable then we should remove them (and put them in the "Attributes" section below)

Reply:
I‘ll change at least the enabled attribute to be configurable. The meaning of the version attribute isn‘t really clear for me. The spec claims that it‘s a template version. Maybe there are several optimized auto scale templates available which can be used instead of the min/max values. 🤷🏼‍♂️